pkg/apis/pipeline/v1alpha1/build_gcs_resource.go

```diff
@@ -0,0 +1,155 @@
+/*
+Copyright 2019 The Knative Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"flag"
+	"fmt"
+	"path/filepath"
+	"strings"
+
+	corev1 "k8s.io/api/core/v1"
+)
+
+var (
+	buildGCSFetcherImage = flag.String("gcs-fetcher-image", "gcr.io/cloud-builders/gcs-fetcher:latest",
+		"The container image containing our GCS fetcher binary.")
+	buildGCSUploaderImage = flag.String("gcs-uploader-image", "gcr.io/cloud-builders/gcs-uploader:latest",
+		"The container image containing our GCS uploader binary.")
```

Comment:
@shashwathi do these need to be flags? I noticed that there's a flag for the gsutil image (https://github.com/knative/build-pipeline/blob/master/pkg/apis/pipeline/v1alpha1/gcs_resource.go#L29) but does that make sense? i think we want to configure these via configmaps if anything :thinking:

Reply:
side note: @tejal29 we should make the fetcher image configurable via configmap, like in https://github.com/knative/build-pipeline/commit/9fd1660aa04f52f4ed8fdfc2fe76e0e8940e07e3

Reply:
Currently all of them are using flags. So if we want to configure via configmaps then I would prefer doing that for all flags instead of one special case.

Reply:
> So if we want to configure via configmaps then I would prefer doing that for all flags instead of one special case.

oh no, i guess that means we already have both, since the image for the entrypoitnt image is configurable via config map :S 

@shashwathi if you agree it makes sense to use configmaps, i can create a separate issue to tackle making them all configurable via config maps

Reply:
I don’t think it needs te be in a configmap because it’s not really expected to change much